vm/actor/src/builtin/market/mod.rs

```diff
@@ -501,7 +502,7 @@ impl Actor {
             let mut lt = BalanceTable::from_root(rt.store(), &st.locked_table)
                 .map_err(|e| ActorError::new(ExitCode::ErrIllegalState, e.into()))?;
 
-            let mut i = st.last_cron + 1;
+            let mut i = st.last_cron.unwrap() + 1;
```

Comment:
can you put a TODO on this to fix, it's going to be changed with Epoch changes, but broken as is